phoenix-core/src/main/java/org/apache/phoenix/expression/LiteralExpression.java

```diff
@@ -242,6 +296,9 @@ private LiteralExpression(Object value, PDataType type, byte[] byteValue,
         this.determinism = deterministic;
     }
 
+    public LiteralExpression () {
```

Comment:
Where is this constructor used?

Reply:
I believe it's used in some of the test cases that access LiteralExpression.<init>
At least, some of the unit tests were failing before I added the default constructor